src/tools/mesh_tester.cpp

```diff
@@ -250,6 +328,109 @@ std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
   return retval;
 }
 
+#if defined(AXOM_USE_RAJA)
+template < typename ExecSpace >
+std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
+  mint::Mesh* surface_mesh,
+  std::vector<int> & degenerate)
+{
+  SLIC_INFO("Running naive intersection algorithm "
+    << " in execution Space: " 
+    << axom::execution_space< ExecSpace >::name());
+
+  // Get allocator
+  #ifdef AXOM_USE_UMPIRE
+    int allocatorID = axom::execution_space< ExecSpace >::allocatorID();
+    umpire::Allocator allocator = axom::getAllocator(allocatorID);
```

Comment:
You should be able to just use `allocatorID` here and get rid of all the `AXOM_USE_UMPIRE` ifdefs